explainability-api/microservice/temp_result/temp_result.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
do we need this file checked-in?

Reply:
I think so, or else 

> with open('temp_result/temp_result.json', 'w') as f

will throw an error.

Reply:
When using `mode="w"`, [open](https://docs.python.org/3/library/functions.html#open) will create a file if it does not exist, or delete it's contents if it does exist. However, it will not create a directory. So you might need to do that first. I would recommend using the builtin `[pathlib](https://docs.python.org/3/library/pathlib.html)` library. Something like this should work:
```python3
from pathlib import Path
output_file = Path(os.environ["OUTPUT_DIR"]) / "result.json"
output_file.parent.mkdir(parents=True, exist_ok=True)
output_file.write_text(json.dumps(result))
```

 How does the service further need this resutl?

Reply:
I think you can change it however you think it best fit. I'm thinking maybe you can simply remove the parts about a new directory and just call open a new json file. Haritz just told us to save the results locally somewhere so we can use it to analyse it locally. So I just saved it and return it through the api. I'm so sorry if i'm asking too much of you :'(